skaffold/README.md

```diff
@@ -0,0 +1,56 @@
+# Skaffold files
+
+Demonstation files for using [skaffold](https://github.com/GoogleContainerTools/skaffold) to develop configurations.
+
+## Purpose
+
+These samples provide developers a faster way to iterate on configurations for the ForgeRock platform. The idea is to perform iteration using skaffold, and then when you are happy with the configuration, publish it to git.  This shortens the development cycle from this:
```

Comment:
Pls use s/ForgeRock platform/ForgeRock Identity Platform/

It's our product name